### PR TITLE
Deleted

### PR DESCRIPTION
**Pull request by Rexadev**

### CODE REVIEW

File: porn [hunk 1]
```diff
@@ -367,3 +367,24 @@
 0.0.0.0 yamidas.yamiboard.com
 0.0.0.0 yamigama.com
 0.0.0.0 yespornplease.com
+#USA
+stripclubguide.com
+thrillist.com
```
Comment (spirillen) [inline porn:372]: The adult stuff, where is that
image
image

Comment (Rexadev) [reply porn:372]: I know some are tourism sites that have pages for where to sin

**Comment by Rexadev** [comment]
@spirillen I also did a similar thing for other places

**Comment by spirillen** [comment]
> @spirillen I have also created pull requests for other places Mexico ,India, Indonesia

All merged here... https://mypdns.org/my-privacy-dns/porn-records/-/merge_requests

As I told you, this is NOT my repo or project, I'm watching (monitoring) it, to pick up useful commits/issues. And in the Spirit of sharing data (matrix.rock's [project](https://mypdns.org/infrastructure/matrix-rocks/www.matrix.rocks)) I do add refs to where I've obtained the data.

What you should not exoect is, that I will copy your PR's to [mypdns.org](https://mypdns.org/) for.
If you expect your records are being blacklisted by mypdns, you'll need to add them your self. I'm not a slave, I'm kindly helps other who believe adult sites are devastating and spending hours, days and even month maintaining the lists in a high standard and not just those bulk commits where half of it, always turns out to be irrelevant or even better [FP](https://mypdns.org/mypdns/support/-/wikis/False-Positive)'s.  
I'm also spending the bucks keeping a tracking free server up and running.

So please, if you think any of these records deserves being blacklisted at mypdns, please do commit them there. There is [various templates](https://mypdns.org/my-privacy-dns/porn-records#submit-contribute) to use for fast commit.

**Comment by Rexadev** [comment]
> > @spirillen I have also created pull requests for other places Mexico ,India, Indonesia

Cool cool, I will submit https://github.com/mypdns/porn-records here, mypdns.org is slow

**Comment by spirillen** [comment]
> > > @spirillen I have also created pull requests for other places Mexico ,India, Indonesia
> 
> Cool cool, I will submit at https://github.com/mypdns/porn-records if you enable "issues", mypdns.org is slow.

That should at no point be slow....

But all issue goes here, https://mypdns.org/my-privacy-dns/porn-records/-/issues

> ... if you enable

I don't support the GH anti-privacy :smirk:
And GH once had been deleting ([taken down](https://mypdns.org/my-privacy-dns/porn-records/-/issues/1347)) that repo that's why I spend the box on running mypdns.org, if you find it slow, instead let me know where you find it slow? and I might be able to tweaks a few things

I do also now understand why you are trying to turn a hosts file into a wildcard list. https://github.com/nextdns/metadata/issues/834

**Comment by Rexadev** [comment]
> > > > @spirillen I have also created pull requests for other places Mexico ,India, Indonesia
....
https://mypdns.org/ only works over US VPN, I tried Asia, Europe

**Comment by spirillen** [comment]
That's very weird... as the firewall actually have been disabled since the 7. for a similar reason

**Comment by spirillen** [comment]
but your right, it fails from romaina, vietnam (tor browser) but allowed from Norway, Netherlands,


Started the firewall and now all seems right :confounded: :confused: 

Please try again @Rexadev 

PS. There are no needs at all to use a VPN for mypdns.... i do not keep any logs at all, once it is rotated, it is deleted.

```
sudo crontab -l
# 0 5 * * 1 tar -zcf /var/backups/home.tgz /home/
# 
# For more information see the manual pages of crontab(5) and cron(8)
# 
# m h  dom mon dow   command
10 * * * * find /var/log/ -type f -name "*.gz" -delete  >/dev/null 2>&1
```

**Pull request closed without merging**